pages/1.11/installing/upgrading/index.md

```diff
@@ -254,7 +282,9 @@ If your upgrade fails because of a [custom node or cluster check](/1.11/installi
 dcos-diagnostics check node-poststart
 dcos-diagnostics check cluster
 ```
-### On DC/OS Masters
+### On DC/OS Masters 
+
+[enterprise type="inline" size="small" /]
```

Comment:
I'm not seeing how the open and enterprise instructions are different from each other here.